### PR TITLE
Factory Plugin

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
Che Theia Factory Plugin
Moved from https://github.com/eclipse/che-theia-factory-extension

### CODE REVIEW

**Comment by sunix** [comment]
do we want to keep git history for these kind of move ?

**Comment by vitaliy-guliy** [comment]
@sunix I'm not sure

**Comment by sunix** [comment]
lets keep like that but the commit comment should say this is been moved from this repo (provide the link)

**Comment by vitaliy-guliy** [comment]
Squashed and changed the commit message

File: plugins/factory-plugin/LICENSE [hunk 1]
```diff
@@ -0,0 +1,277 @@
+Eclipse Public License - v 2.0
```
[the remark this answers is not in the document]
Comment (vitaliy-guliy) [reply plugins/factory-plugin/LICENSE]: Moved to repository root

File: plugins/factory-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
+[![Build Status](https://travis-ci.org/eclipse/che-theia-factory-extension.svg?branch=master)](https://travis-ci.org/eclipse/che-theia-factory-extension)
```
[the remark this answers is not in the document]
Comment (vitaliy-guliy) [reply plugins/factory-plugin/README.md]: I suppose we need to update travis

**Pull request merged**